bundle/bundle.go

```diff
@@ -53,6 +53,7 @@ type Bundle struct {
 // SignaturesConfig represents an array of JWTs that encapsulate the signatures for the bundle.
 type SignaturesConfig struct {
 	Signatures []string `json:"signatures,omitempty"`
+	Plugin     string   `json:"plugin,omitempty"`
```

Comment:
This means if we have multiple signatures (currently we don't) we'll use the same verifier for all the signatures ? Need to think about this out a bit.

Reply:
i'm open to suggestions.  imo, it seems unlikely there would be a need to have multiple signing/verification implementations in a single bundle.  something that mapped signatures to plugins would probably be a breaking change to `.signatures.json`, so maybe we roll with this until there's actually a need for multiple signatures and signing/verification implementations?